consensus/istanbul/backend/kip103.go

```diff
@@ -42,8 +42,12 @@ func (caller *Kip103ContractCaller) CallContract(ctx context.Context, call klayt
 	// call.To: the target contract address will be assigned by `BoundContract`
 	// call.Value: nil value is acceptable for `types.NewMessage`
 	// call.Data: a proper value will be assigned by `BoundContract`
+	var accessList types.AccessList
+	if call.AccessList != nil {
+		accessList = *call.AccessList
+	}
 	msg := types.NewMessage(call.From, call.To, caller.state.GetNonce(call.From),
-		call.Value, gasLimit, gasPrice, call.Data, false, intrinsicGas)
+		call.Value, gasLimit, gasPrice, call.Data, false, intrinsicGas, accessList)
```

Comment:
Do we need to pass `accessList` instead of `nil`?

Reply:
I'm not sure if caller of `Kip103ContractCaller` would specify access list when calling `CallContract`. But wouldn't it be OK to properly handle access list if it was specified?

Reply:
I agree that it would be better not to handle access list here because:
- it's a call so handling access list won't have any effect
- the caller of `Kip103ContractCaller` probably won't use access list

Updated, PTAL